docs/Developer-guide/Backport-Guidelines.md

```diff
@@ -1,18 +1,18 @@
 # Backport Guidelines
 In GlusterFS project, as a policy, any new change, bug fix, etc., are to be
-fixed in 'master' branch before release branches. When a bug is fixed in
-the master branch, it might be desirable or necessary in release branch.
+fixed in 'devel' branch before release branches. When a bug is fixed in
```

Comment:
are you going to name this branch to devel or main? github recommends it to be main though...

Reply:
At present, the default branch of glusterfs repo is `devel`, I think that was done to indicate that it is a continuous development branch. But yes, github does recommend `main` and even the default branch of glusterdocs is named `main`. @sankarshanmukhopadhyay @amarts  any thoughts on this ?

Reply:
If this is the same topic we had a brief interaction on the Slack channel then I'd opt for main and devel. The former being a replacement for the term 'master'

Reply:
For glusterfs repo, changing branch name again would be an issue. So, this document is fine. For all other projects which are not gone through branch changes, I am fine with `main` as in future the default branch would be `main` in all repos.